### ISSUE TITLE
[Bug Report] My background stopped updating to my image.

### ISSUE THREAD

**Issue by EdgarNonsense** [shalldie/vscode-background#446]
## Summary | 概述

I recently reopened VSCode to see that the background that I chose had stop working.

- `OS - Win10 `:
- `vscode`: 1.88

### Screenshots | 截图
![image](https://github.com/shalldie/vscode-background/assets/70247767/d28f1af2-5e81-4161-9ed6-5d9328e10f95)

Basically image isn't going from the default to what I wanted.

**Comment by shalldie** [comment]
What's your config?

**Comment by EdgarNonsense** [comment]
> What's your config?

Ryzen 7 7800x3D
32 gigs of ram
3080TI

**Comment by EdgarNonsense** [comment]
` "background.enabled": true,
    "background.fullscreen": {
      "image": "https://preview.redd.it/7uswogiku7mb1.jpg?width=960&crop=smart&auto=webp&s=554101e31dee638762b111c34d251d24d6925c8e", // url of your image
      "opacity": 0.95, // 0.85 ~ 0.95 recommended
      "size": "cover", // also css, `cover` to self-adaption (recommended)，or `contain`、`200px 200px`
      "position": "center", // alias to `background-position`, default `center`
      "interval": 0 // seconds of interval for carousel, default `0` to disabled.
    },`

**Comment by suiyun39** [comment]
First, please use `images` and not `image`.

Second, the url netowrk is blocked, must login reddit account or use developer token, please pay special attention to this issue.

**Comment by EdgarNonsense** [comment]
Updated to this

"background.enabled": true,
    "background.fullscreen": {
      "images": "https://images.unsplash.com/photo-1519638399535-1b036603ac77?q=80&w=1000&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8fA%3D%3D", // url of your image
      "opacity": 0.95, // 0.85 ~ 0.95 recommended
      "size": "cover", // also css, `cover` to self-adaption (recommended)，or `contain`、`200px 200px`
      "position": "center", // alias to `background-position`, default `center`
      "interval": 0 // seconds of interval for carousel, default `0` to disabled.
    },

issue still persists

**Comment by shalldie** [comment]
@EdgarNonsense 

`images` is an Array, `string[]` type. [referrence](https://github.com/shalldie/vscode-background?tab=readme-ov-file#fullscreen-config)

**Comment by EdgarNonsense** [comment]
Nah, a different issue pulled up, so I kinda gave up on it, it's not even displaying images anymore, I'm assuming it's a conflict from my other extensions

**Comment by shalldie** [comment]
> Nah, a different issue pulled up, so I kinda gave up on it, it's not even displaying images anymore, I'm assuming it's a conflict from my other extensions

@EdgarNonsense 
Sry to hear that.
If you are willing to try again, you can send us your latest configuration to take a look

**Comment by shalldie** [comment]
```json
  "background.enabled": true,
  "background.fullscreen": {
    "images": [
      "https://images.unsplash.com/photo-1519638399535-1b036603ac77?q=80&w=1000&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8fA%3D%3D"
    ], // url of your image
    "opacity": 0.90, // 0.85 ~ 0.95 recommended
    "size": "cover", // also css, cover to self-adaption (recommended)，or contain、200px 200px
    "position": "center", // alias to background-position, default center
    "interval": 0 // seconds of interval for carousel, default 0 to disabled.
  }
```

This works fine for me.

**Comment by EdgarNonsense** [comment]
Huh, okay yeah your thing works, i guessing there's probably an issue with my image or something
